release/bin/startup.bat

```diff
@@ -7,10 +7,8 @@ del logs\*request.log*
 del logs\output.log
 move logs\geonetwork.log.* logs\archive
 
-java -Dmime-mappings=..\web\geonetwork\WEB-INF\mime-types.properties -DSTOP.PORT=8079 -Djava.awt.headless=true -DSTOP.KEY=geonetwork -jar start.jar
+java -Xms512m -Xmx1g -Djetty.httpConfig.requestHeaderSize=32768 -Dorg.eclipse.jetty.server.Request.maxFormContentSize=500000 -Dorg.eclipse.jetty.server.Request.maxFormKeys=4000 -Dmime-mappings=..\web\geonetwork\WEB-INF\mime-types.properties -DSTOP.PORT=8079 -Djava.awt.headless=true -DSTOP.KEY=geonetwork -jar start.jar
```

Comment:
Are these changes related to the pull request or are more general changes?

Reply:
This is to have a more consistent configuration between docker images https://github.com/geonetwork/docker-geonetwork/pull/107 and the ZIP with Jetty.